pro/src/components/pages/Offerers/Offerer/VenueV1/VenueEdition/ImageVenueUploaderSection/ImageVenueUploaderSection.tsx

```diff
@@ -18,12 +18,16 @@ import { VenueImageUploaderModal } from './VenueImageUploaderModal'
 type Props = {
   venueId: string
   venueImage: string | null
+  venueCredit: string
+  setVenueCredit: (credit: string) => void
   children?: never
 }
 
 export const ImageVenueUploaderSection: FunctionComponent<Props> = ({
   venueId,
   venueImage,
+  venueCredit,
+  setVenueCredit,
 }) => {
```

Comment:
Ca n'est pas la syntax que nous utilisons:
```suggestion
export const ImageVenueUploaderSection: ({
  venueId,
  venueImage,
  venueCredit,
  setVenueCredit,
}): IImageVenueUploaderSectionProps => {
```